server/pregnancy-centers/schema/joi-schema.js

```diff
@@ -60,7 +60,7 @@ const pregnancyCenterSchemaJoi = Joi.object().keys({
 	name: Joi.string(), // change to PRC name
 	notes: Joi.string(),
 	phone: phoneValidator.phone().validate(),
-	primaryContactUserId: Joi.string(),
+	primaryContactPersonId: Joi.string(),
```

Comment:
This should actually be an objectId. You'll need to import it from mongo, like this:

```
const mongoose = require('mongoose')
const ObjectId = mongoose.Schema.Types.ObjectId
```